src/main/java/com/epam/tc/hw/mobile/pageObjects/WebPageObject.java

```diff
@@ -10,22 +10,33 @@
 
 import java.util.LinkedList;
 import java.util.List;
+import java.util.Objects;
 
 public class WebPageObject  {
 
     @FindBy(xpath = ".//input[@name='q']")
-    WebElement googleInput;
+    private WebElement googleInput;
 
     @FindBy(xpath = ".//a[@class='cz3goc BmP5tf']")
-    List<WebElement> googleSearch;
+    private List<WebElement> googleSearch;
+
+    @FindBy(xpath = "(.//*[@class= 'sbic sb43'])[1]")
+    private WebElement iosFirstLink;
 
     public WebPageObject(AppiumDriver<?> appiumDriver) {
         PageFactory.initElements(appiumDriver, this);
+        new WebDriverWait(appiumDriver, 10).until(
+                wd -> ((JavascriptExecutor) wd).executeScript("return document.readyState").equals("complete")
+        );
     }
 
-    public void searchOnGoogle(String input) {
+    public void searchOnGoogle(String input, String platform) {
         googleInput.sendKeys(input);
-        googleInput.sendKeys(Keys.ENTER);
+         if (Objects.equals(platform, "Android")) {
```

Comment:
It's better not to keep strings in the code, use constants or create your own enum to manipulate the range of similar strings